system/jlib/jsecrets.cpp

```diff
@@ -438,6 +449,22 @@ class CVault
 
     CVaultKind getVaultKind() const { return kind; }
 
+    void initClient(httplib::Client &cli, httplib::Headers &headers, unsigned &numRetries)
+    {
+        numRetries = retries;
+        cli.enable_server_certificate_verification(verify_server);
+        if (connectTimeout)
```

Comment:
Are seconds the correct granularity for setting timeouts?  Should it be using us instead?